static/lang/re-en.json

```diff
@@ -2386,6 +2386,17 @@
                 "Cooldown": "Your eidolon then can't use their Breath Weapon again for the next [[/r 1d4 #rounds]]{1d4 rounds}.",
                 "Scaling": "At 3rd level and every 2 levels thereafter, the damage increases by 1d6."
             },
+            "BrokenChain": {
+                "ReleaseMe": {
+                    "BonusToggle": "Release Me! — Apply melee Strike bonus"
+                },
+                "UltimatumOfLiberation": {
+                    "BonusIncreaseToggle": "Ultimatum of Liberation — Increase save bonus to +3"
```

Comment:
Should this be `"Ultimatum of Liberation — Within 30 feet of origin"`? I could go either way I guess.

Reply:
I considered something along those lines but I couldn't get something unambiguous enough that also wasn't super long.